### PR TITLE
Clarify addlocked description in getbalance RPC

### PR DESCRIPTION
**Pull request by thephez**

### CODE REVIEW

File: src/wallet/rpcwallet.cpp [hunk 1]
```diff
@@ -772,7 +772,7 @@ UniValue getbalance(const JSONRPCRequest& request)
             "\nArguments:\n"
             "1. \"account\"        (string, optional) DEPRECATED. The selected account, or \"*\" for entire wallet. It may be the default account using \"\".\n"
             "2. minconf          (numeric, optional, default=1) Only include transactions confirmed at least this many times.\n"
-            "3. addlocked      (bool, optional, default=false) Whether to add  balance from transactions locked via InstantSend.\n"
+            "3. addlocked      (bool, optional, default=false) Whether to add the balance from transactions locked via InstantSend.\n"
             "4. include_watchonly (bool, optional, default=false) Also include balance in watch-only addresses (see 'importaddress')\n"
```
Comment (Duality-CDOO) [inline src/wallet/rpcwallet.cpp]: Wouldn't you want to be consistent and have "Also include the balannce in watch-only addresses (see 'importaddress')\n"

Comment (thephez) [reply src/wallet/rpcwallet.cpp]: Perhaps, but that is not consistent looking across all the RPCs (e.g. [listreceivedbyaddress](https://github.com/dashpay/dash/blob/develop/src/wallet/rpcwallet.cpp#L1300)). I just noticed the double-space and corrected it. Making them all consistent would be a bigger task and possibly complicate future backports from Bitcoin. 
@UdjinM6 @PastaPastaPasta What do you guys think?

Comment (Duality-CDOO) [reply src/wallet/rpcwallet.cpp]: I agree that this change is rather trivial and poses more of an issue at upstream than what the change warrants.

Comment (PastaPastaPasta) [reply src/wallet/rpcwallet.cpp]: Imo non dash specific rpcs should be keep in line with upstream unless there is a substancial reason, ie not trivial.

File: src/wallet/rpcwallet.cpp [hunk 2]
```diff
@@ -772,7 +772,7 @@ UniValue getbalance(const JSONRPCRequest& request)
             "\nArguments:\n"
             "1. \"account\"        (string, optional) DEPRECATED. The selected account, or \"*\" for entire wallet. It may be the default account using \"\".\n"
             "2. minconf          (numeric, optional, default=1) Only include transactions confirmed at least this many times.\n"
-            "3. addlocked      (bool, optional, default=false) Whether to add  balance from transactions locked via InstantSend.\n"
+            "3. addlocked      (bool, optional, default=false) Whether to add the balance from transactions locked via InstantSend.\n"
```
[the remark this answers is not in the document]
Comment (UdjinM6) [reply src/wallet/rpcwallet.cpp]: I like the second one

**Pull request merged**